blocks/filter.py

```diff
@@ -96,6 +96,16 @@ class object contained in the list.
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
                  name_regex=None, applications=None):
+        if bricks is not None and not isinstance(bricks, Brick):
```

Comment:
I guess that `isclass` can be avoided here https://github.com/bartvm/blocks/blob/67e19f33ccdd2d21482fcfce89542a60c0385bad/blocks/filter.py#L131

Also, I am not sure if something can be changed here: https://github.com/bartvm/blocks/blob/67e19f33ccdd2d21482fcfce89542a60c0385bad/blocks/filter.py#L147-L150